runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsPlugin.java

```diff
@@ -24,22 +24,48 @@
  */
 package net.runelite.client.plugins.boosts;
 
+import com.google.common.collect.ObjectArrays;
+import com.google.common.eventbus.Subscribe;
 import com.google.inject.Binder;
 import com.google.inject.Provides;
 import javax.inject.Inject;
+import lombok.Getter;
+import net.runelite.api.Skill;
+import net.runelite.api.events.ConfigChanged;
 import net.runelite.client.config.ConfigManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
 
 @PluginDescriptor(
 	name = "Boosts plugin"
 )
 public class BoostsPlugin extends Plugin
 {
+	private static final Skill[] COMBAT = new Skill[]
+	{
+			Skill.ATTACK, Skill.STRENGTH, Skill.DEFENCE, Skill.RANGED, Skill.MAGIC
+	};
+	private static final Skill[] SKILLING = new Skill[]
+	{
+			Skill.MINING, Skill.AGILITY, Skill.SMITHING, Skill.HERBLORE, Skill.FISHING, Skill.THIEVING,
+			Skill.COOKING, Skill.CRAFTING, Skill.FIREMAKING, Skill.FLETCHING, Skill.WOODCUTTING, Skill.RUNECRAFT,
+			Skill.SLAYER, Skill.FARMING, Skill.CONSTRUCTION, Skill.HUNTER
+	};
+
+	@Inject
+	InfoBoxManager infoBoxManager;
+
 	@Inject
 	BoostsOverlay boostsOverlay;
 
+	@Inject
+	BoostsConfig config;
```

Comment:
Make the new injected fields private (and adjust existing injected fields in the file to be private as well).